HSStockChart/TimeLineChart/HSTimeLine.swift

```diff
@@ -107,6 +122,25 @@ public class HSTimeLine: UIView, HSDrawLayerProtocol {
         self.addGestureRecognizer(tapGesture)
     }
     
+    //Getter Setter for customization
+    public func setCountOfTimes(times: Int)
+    {
+        self.countOfTimes = times
+    }
+    
+    public func setfiveDayCount(counts: Int)
+    {
+        self.fiveDayCount = counts
+    }
```

Comment:
Change `countOfTimes` and `fiveDayCount` to `public var` is OK. There is no need to make setter function.

Reply:
```suggestion
  
```